### ISSUE TITLE
Issue with SSL: required for all webhooks, but when it's plugged into Flask examples, it doesn't work

### ISSUE THREAD

**Issue by wfn** [line/line-bot-sdk-python#84]
Hello,

Current Line API appears to require SSL for all webhooks (i.e. not just webhook URL verification). When I set up a working SSL context for Flask (see below), with a valid certificate chain from Let's Encrypt (root CA ISRG which is trusted by Line API), nothing appears to happen.

If I launch the examples (e.g. Flash echo bot) as is, I get HTTPS negotiation requests from Line API (Flask/Werkzeug complains as it's basically unexpected binary data, and can't handle it). However, if I set up a working HTTPS with a valid certificate (not self-signed), no webhook events are reported - I think Line API drops the connection since it is not satisfied with the HTTPS negotiation, but I'm yet to find a way for Werkzeug/whatever to debug-level report me all connection initiation attempts and how they look like:

```
        import ssl
        ctx = ssl.SSLContext(ssl.PROTOCOL_SSLv23)
        ctx.load_cert_chain('pb1.mkj.lt/cert.pem', 'pb1.mkj.lt/privkey.pem')
        app.run(debug=options.debug, host="0.0.0.0", port=int(options.port), ssl_context=ctx)
```

This runs fine, but no webhooks are reported from the Line API. The webhook URL verification on Line Manager also fails without further explanation.

How should this be done, and is there a way to debug SSL issues / get info from Line API what fails in particular?

Thank you!
Kostas

_P.S._ I've also just tried with `ssl.PROTOCOL_TLSv1_2` - same result.

**Comment by wfn** [comment]
Oh, it appears that I may have found what's wrong with my code>

1. In `load_cert_chain` call, `fullchain.pem` should be passed instead of `cert.pem`.
2. "/callback" should be added to the webhook URL on Line Manager.

However, I am now getting `linebot.exceptions.LineBotApiError: <LineBotApiError [Invalid reply token]>` with the flask echo example. I checked to make sure that the reply token posted back is the one which had been attached to the message received. Should I assume that this token is bogus, though? i.e., this means all is fine?

Thanks for reading. :)

**Comment by wfn** [comment]
Closing this issue as I've sorted this myself and this has fully been an issue I had created myself. TL;DR for other devs experiencing similar issues is [the comment immediately above](https://github.com/line/line-bot-sdk-python/issues/84#issuecomment-351754981), i.e., if you use a proper valid full certificate chain, all is well. Make sure to add "/callback" to the webhook URL setting in Line Manager.

And to answer my last question above, yes, the reply token in the webhook validation function in Line Manager is bogus (i.e. reply token given by the API is invalid), and that is fine.
